test/e2e/formatted_error_test.go

```diff
@@ -50,16 +50,13 @@ spec:
 
 	logger.Section("deploy with errors", func() {
 		expectedErr := strings.TrimSpace(`
-kapp: Error: Applying create job/successful-job (batch/v1) namespace: default:
-  Creating resource job/successful-job (batch/v1) namespace: default:
-    API server says:
-      Job.batch "successful-job" is invalid: 
+kapp: Error: Finished unsuccessfully: [create job/successful-job (batch/v1) namespace: default: Creating resource job/successful-job (batch/v1) namespace: default: API server says: Job.batch "successful-job" is invalid: 
```

Comment:
do we not get nicely formatted messages anymore?

Reply:
Apparently not when there is only 1 error in the array. I will take a closer look at `SemiStructuredError`.